table/tables/tables_test.go

```diff
@@ -392,3 +392,94 @@ func (ts *testSuite) TestTableFromMeta(c *C) {
 	_, err = tb.AllocHandle(tk.Se)
 	c.Assert(err, NotNil)
 }
+
+func (ts *testSuite) TestHiddenColumn(c *C) {
+	tk := testkit.NewTestKit(c, ts.store)
+	tk.MustExec("CREATE DATABASE test_hidden;")
```

Comment:
`drop database if exists` firstly, or use`create database if not exists`

Reply:
Done.